README.md

```diff
@@ -41,59 +45,109 @@ We recommend that you use `@dhis2/ui` as the entrypoint for all imports of our f
 
 ### `@dhis2/ui-constants`
 
-This package provides access to shared constants, such as colors, spacers and elevation values. They are used across our frontend components and can be used directly in applications as well. Our constants can be imported like so:
+This package provides access to shared constants, such as colors,
+spacers and elevation values. They are used across our frontend
+components and can be used directly in applications as well. Our
+constants can be imported like so:
 
 ```js
 import { colors } from '@dhis2/ui'
 ```
 
-See our [api docs](https://ui.dhis2.nu/#/api) for a full list of the available constants.
+See our [api docs](https://ui.dhis2.nu/#/api) for a full list of the
+available constants.
+
+### `@dhis2/ui-elements`
+
+Think of these components as styled HTML elements. These are the
+elements that are used to compose other UI components.
+
+-   Elements MUST NOT import other elements.
+-   Elements SHOULD NOT provide behavior.
+-   Elements MUST provide relevant styles for the design system.
+-   Elements MUST forward the `ref`.
+-   Elements MAY provide props for specific variants.
+-   Elements SHOULD spread props to the underlying HTML element.
+
+Since UI Elements are useful in other contexts to compose new components
+with specific behavior, they provide a stable base to build on with a
+flexible API.
```

Comment:
:point_up: This is an attempt at defining UI elements using keywords as defined by: https://tools.ietf.org/html/rfc2119

We might be able to strengthen the `MAY` to `SHOULD` and the `SHOULD`s to `MUST`s.